Formula/lib/libaec.rb

```diff
@@ -19,21 +20,18 @@ class Libaec < Formula
 
   depends_on "cmake" => :build
 
-  # These may have been linked by `szip` before keg_only change
-  link_overwrite "include/szlib.h"
-  link_overwrite "lib/libsz.a"
-  link_overwrite "lib/libsz.dylib"
-  link_overwrite "lib/libsz.2.dylib"
-  link_overwrite "lib/libsz.so"
-  link_overwrite "lib/libsz.so.2"
-
   def install
     # run ctest for libraries, also added `"-DBUILD_TESTING=ON` in the end as
     # `std_cmake_args` has `BUILD_TESTING` off
     system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DBUILD_TESTING=ON"
     system "cmake", "--build", "build"
     system "ctest", "--test-dir", "build", "--verbose"
     system "cmake", "--install", "build"
+
+    # Move cmake files to common linked location similar to Linux distros
+    # Ref: https://gitlab.archlinux.org/archlinux/packaging/packages/libaec/-/blob/main/PKGBUILD?ref_type=heads#L25
+    # Ref: https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/community/libaec/APKBUILD#L43
+    lib.install prefix/"cmake"
```

Comment:
Could do this to avoid breaking things.
```suggestion
    lib.install_symlink prefix/"cmake"
```